### PR TITLE
Updated Sansay info

### PR DESCRIPTION
**Pull request by FilippSe**
Updated Sansay info

### CODE REVIEW

File: Teams/direct-routing-border-controllers.md [hunk 1]
```diff
@@ -95,7 +95,8 @@ Note that we are not accepting new nominations for certification until further n
 |[Cataleya](https://cataleya.com/orchidplatforms/)|Orchid Link|&#10004;||3.1|||
 |[ULTATEL](https://www.ultatel.com/services/direct-routing-teams-sbc)|Teams SBC|&#10004;|&#10004;|1.6|||
 |[Atos](https://unify.com/en/solutions/voice-platforms/session-border-controller)|Atos Unify OpenScape Session Border Controller|&#10004;|&#10004;|V10R2.2.0|||
-|[Sansay Inc.](https://www.sansay.com/solutions/microsoft-teams/)|vmVSXi|&#10004;|&#10004;|10.5.1.354-vm-S-x64|&#10004;||
+|[Sansay Inc.](https://www.sansay.com/solutions/microsoft-teams/)|VSXi-10.7.1.80|&#10004;|&#10004;|Supported on versions 10.5.1.354, 10.7.1.x and 11.1.1.x. (Recommended: 11.1.1.x)|&#10004;||
+||VSXi-10.5.1.354|&#10004;|&#10004;|Supported on versions 10.5.1.354, 10.7.1.x and 11.1.1.x. (Recommended: 11.1.1.x)|&#10004;||
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-border-controllers.md:99]:
```suggestion
|[Sansay Inc.](https://www.sansay.com/solutions/microsoft-teams/)|VSXi-10.7.1.80|&#10004;|&#10004;|Supported on versions 10.5.1.354, 10.7.1.x, and 11.1.1.x. (Recommended 11.1.1.x)|&#10004;||
||VSXi-10.5.1.354|&#10004;|&#10004;|Supported on versions 10.5.1.354, 10.7.1.x, and 11.1.1.x. (Recommended 11.1.1.x)|&#10004;||
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @FilippSe, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
@scanum, @FilippSe :  I am unable to merge this PR.  Please see the notice above:  Merging is blocked because the base branch should be public?  and not live?

**Comment by tiburd** [comment]
@FilippSe 
Hi,

Please change the branch this PR is to merge to from "live" to "public"; it should be the default branch:
image

Also, please note that this pull request was opened in the public repo. PMs and content developers should work in the private repo, as described in [the authoring guide](https://review.docs.microsoft.com/en-us/Office-Authoring-Guide/office-docs-authoring-overview?branch=main). If you prefer, you can close this PR and submit this change via the private repo instead of changing the branch in this PR.

We encourage you to use the private repo for future contributions to Microsoft Learn.

Thank you!

Thank you!

**Comment by dstrome** [comment]
If changes in this PR are still required, please submit a new PR

**Pull request closed without merging**